schema/settings.json

```
@@ -41,6 +41,12 @@
       "description": "Show the current execution time (last execution time) as a cell executes.",
       "default": true
     },
+    "showDate": {
+      "type": "boolean",
+      "title": "Show Last Execution Date",
+      "description": "Show the date in browser timezone when the cell was last executed.",
```

Since the date format is now customizable (#93), it is not necessarily in browser timezone anymore:

```suggestion
      "description": "Show the date of last cell execution.",
```